mat64/format.go

```diff
@@ -11,13 +11,14 @@ import (
 
 // Format prints a pretty representation of m to the fs io.Writer. The format character c
 // specifies the numerical representation of of elements; valid values are those for float64
-// specified in the fmt package, with their associated flags. In addition to this, a '#' for
-// all valid verbs except 'v' indicates that zero values be represented by the dot character.
-// The '#' associated with the 'v' verb formats the matrix with Go syntax representation.
+// specified in the fmt package, with their associated flags. In addition to this, a space
+// preceding a verb indicates that zero values should be represented by the dot character.
 // The printed range of the matrix can be limited by specifying a positive value for margin;
 // If margin is greater than zero, only the first and last margin rows/columns of the matrix
 // are output.
-func Format(m Matrix, margin int, dot byte, fs fmt.State, c rune) {
+//
+// Format will not provide Go syntax output.
```

Comment:
The central issue here is that Format is really a helper for people who want to provide fmt.Formatter support for their matrix type (nominally us when we sort out the wrinkles, but also others). The problem is that fmt.Formatter is very poorly documented (for the full extent of the docs see [here](http://golang.org/pkg/fmt/#Formatter) and [here](http://golang.org/pkg/fmt/#State)). As I wrote in the golang-dev thread that sparked this, Format is really a stub that people can use to do all the work of formatting a matrix with minimal work on their part - and uniformly across all types. In short, people should actually not care at all what fs or c are.